account_move_reversal_choose_method/wizard/account_move_reversal.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 ForgeFlow S.L. (http://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountMoveReversal(models.TransientModel):
+    _inherit = "account.move.reversal"
+
+    refund_method = fields.Selection(
+        selection=[
+            ("refund", "Partial Refund / Reversal in draft without reconciliation"),
+            ("cancel", "Full Refund / Reversal with reconciliation"),
+            ("modify", "Full refund / reversal and new draft invoice / journal entry"),
```

Comment:
Why don't leave the selection naming as the standard (https://github.com/odoo/odoo/blob/0a52c2166b7cffd39851cf329736ac76487a150f/addons/account/wizard/account_move_reversal.py#L18)? You already have a proper description in the wizard:

![image](https://user-images.githubusercontent.com/71635103/124547161-00e70700-de2c-11eb-8cbb-e19de84726bc.png)

Reply:
Good point. @MiquelRForgeFlow can you revert?

Reply:
Not sure. That side description is only for _in_invoice_ and _out_invoice_. If I enable them through xpath, then I would need to edit likewise those descriptions, and that would be a lot of more complex xpaths. I think it's not worth the effort.

![Selection_164](https://user-images.githubusercontent.com/25005517/124572974-65fd2580-de49-11eb-9e41-0b68b9812827.png)

Reply:
IMO I would prefer adding the side description in the view rather than overriding the selection field.

Reply:
Done.